### PR TITLE
fix(web.private-database): limit innodb_buffer_pool_size choices

### PR DESCRIPTION
**Pull request by Meuh-42**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | PUDBLOW-1826
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related
:flags: Translation:
- [x] TRANS-52566

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Can you please include ticket reference in the commit description?

**Comment by Meuh-42** [comment]
@anooparveti 

Done as requested

**Review by anooparveti — DISMISSED** [review]

File: packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.controller.js [hunk 1]
```diff
@@ -83,6 +83,21 @@ angular.module('App').controller(
           };
         });
 
+        field.selectedValue = find(field.availableValues, {
+          id: field.value,
+        });
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.controller.js:88]: we avoid to use external functions if native one could do it.
```suggestion
        field.selectedValue = field.availableValues.find((value) => value.id === field.value);
```

Comment (Meuh-42) [reply packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.controller.js]: Suggestion applied on whole file.

**Review by anooparveti — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**